op-challenger/game/fault/types/position.go

```diff
@@ -146,24 +133,12 @@ func (p Position) ToGIndex() *big.Int {
 
 // bigMSB returns the index of the most significant bit
 func bigMSB(x *big.Int) int {
-	if x.Cmp(common.Big0) == 0 {
+	if x.Cmp(new(big.Int)) == 0 {
```

Comment:
Let's go with `big.newInt(0)` here since the 0 value is important.  Sounds like we should avoid `common.Big0` given its mutable and may randomly become non-zero on us.
```suggestion
	if x.Cmp(big.NewInt(0)) == 0 {
```